java/src/main/java/com/logicalclocks/hsfs/FeatureGroup.java

```diff
@@ -211,6 +217,43 @@ public void updateStatisticsConfig() throws FeatureStoreException, IOException {
     featureGroupEngine.updateStatisticsConfig(this);
   }
 
+  /**
+   * Update the description of the feature group.
+   *
+   * @param description
+   * @throws FeatureStoreException
+   * @throws IOException
+   */
+  public void updateDescription(String description) throws FeatureStoreException, IOException {
+    featureGroupEngine.updateDescription(this, description);
+  }
+
+  /**
+   * Append features to the schema of the feature group.
+   * It is only possible to append features to a feature group. Removing features is considered a breaking change.
+   *
+   * @param features
+   * @throws FeatureStoreException
+   * @throws IOException
+   */
+  public void appendFeatures(List<Feature> features) throws FeatureStoreException, IOException {
+    featureGroupEngine.appendFeatures(this, new ArrayList<>(features));
```

Comment:
Why the `ArrayList<>` here? You already have a list.

Reply:
that list is immutable for some reason or didn't have the `addAll` method